rho/facthashcommand.py

```diff
@@ -7,33 +7,41 @@
 # along with this software; if not, see
 # http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
 
-"""Encrypt sensitive facts."""
+"""Hash sensitive facts."""
 
 from __future__ import print_function
 
 import csv
 import sys
 import os
 from optparse import SUPPRESS_HELP  # pylint: disable=deprecated-module
+import hashlib
 from rho import utilities
 from rho.clicommand import CliCommand
 from rho.translation import _
 from rho.utilities import multi_arg, _read_in_file, write_csv_data
-from rho.vault import get_vault
 
-PROMPT = "Please enter your encryption password: "
+
+def compute_md5_hash(input_string):
```

Comment:
MD5 is not a cryptographically secure hash function any more. Not that this is an incredibly high-security application, but let's use SHA-256 (or something similar), which is guaranteed to be in `hashlib`.

Reply:
@mdvickst Can you weigh in on this? You had specifically mentioned MD5 so I want to implement the correct thing.

Reply:
I'm fine with SHA-256.